cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -43,11 +54,30 @@ public Publication generatePublication() {
                    .withPublisher(extractOrganization())
                    .withOwner(cristinObject.getPublicationOwner())
                    .withStatus(PublicationStatus.DRAFT)
+                   .withFileSet(createFileSetPointingToTheNilFile())
+                   .build();
+    }
+
+    private static File createNonExistentFile() {
```

Comment:
Turns out FE doesn't use URIs for llicenses either… :( should probably create a ticket for that…